metron-analytics/metron-profiler/README.md

```diff
@@ -204,21 +204,21 @@ The Profiler runs as an independent Storm topology.  The configuration for the P
-| profiler.period.duration | The duration of each profile period.  This value should be defined along with `profiler.period.duration.units`.  |
-| profiler.period.duration.units | The units used to specify the `profiler.period.duration`. |
-| profiler.ttl | If a message has not been applied to a Profile in this period of time, the Profile will be forgotten and its resources will be cleaned up. This value should be defined along with `profiler.ttl.units`. |
-| profiler.ttl.units | The units used to specify the `profiler.ttl`. |
-| profiler.hbase.salt.divisor  |  A salt is prepended to the row key to help prevent hotspotting.  This constant is used to generate the salt.  Ideally, this constant should be roughly equal to the number of nodes in the Hbase cluster.  |
-| profiler.hbase.table | The name of the HBase table that profiles are written to.  |
-| profiler.hbase.column.family | The column family used to store profiles. |
-| profiler.hbase.batch | The number of puts that are written in a single batch.  |
-| profiler.hbase.flush.interval.seconds | The maximum number of seconds between batch writes to HBase. |
+| Setting                               | Description
+|---                                    |---
+| profiler.workers                      | The number of worker processes to create for the topology.
+| profiler.executors                    | The number of executors to spawn per component.
+| profiler.input.topic                  | The name of the Kafka topic from which to consume data.
+| profiler.output.topic                 | The name of the Kafka topic to which profile data is written.  Only used with profiles that use the [`triage` result field](#result).
+| profiler.period.duration              | The duration of each profile period.  This value should be defined along with `profiler.period.duration.units`.
+| profiler.period.duration.units        | The units used to specify the `profiler.period.duration`.
+| profiler.ttl                          | If a message has not been applied to a Profile in this period of time, the Profile will be forgotten and its resources will be cleaned up. This value should be defined along with `profiler.ttl.units`.
+| profiler.ttl.units                    | The units used to specify the `profiler.ttl`.
+| profiler.hbase.salt.divisor           | A salt is prepended to the row key to help prevent hotspotting.  This constant is used to generate the salt.  Ideally, this constant should be roughly equal to the number of nodes in the Hbase cluster.
+| profiler.hbase.table                  | The name of the HBase table that profiles are written to.
+| profiler.hbase.column.family          | The column family used to store profiles.
+| profiler.hbase.batch                  | The number of puts that are written in a single batch.
+| profiler.hbase.flush.interval.seconds | The maximum number of seconds between batch writes to HBase.
```

Comment:
I don't understand why you removed the terminal "|" from each line.  Per https://help.github.com/articles/organizing-information-with-tables/ the correct format is to have them.  And it works fine in doxia.  Same in other places in patch.